portal-web/test/functional/com/liferay/portalweb/macros/MySecondMacro.macro

```diff
@@ -0,0 +1,26 @@
+<definition>
```

Comment:
The macro commands below are fine for this exercise, but in the future please note that all related macros should be stored in separate files (i.e. you likely won't have a `navigateToBlogs` command in the same file as an `addTagList`)